src/main/java/uk/gov/ons/census/fwmt/jobservice/service/routing/hh/HHCancelledPause.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.ons.census.fwmt.jobservice.service.routing.hh;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.census.fwmt.common.error.GatewayException;
+import uk.gov.ons.census.fwmt.common.rm.dto.ActionInstructionType;
+import uk.gov.ons.census.fwmt.common.rm.dto.FwmtActionInstruction;
+import uk.gov.ons.census.fwmt.events.component.GatewayEventManager;
+import uk.gov.ons.census.fwmt.jobservice.data.GatewayCache;
+import uk.gov.ons.census.fwmt.jobservice.service.processor.InboundProcessor;
+import uk.gov.ons.census.fwmt.jobservice.service.processor.ProcessorKey;
+
+import java.time.Instant;
+
+@Qualifier("Pause")
+@Service
+public class HHCancelledPause implements InboundProcessor<FwmtActionInstruction> {
+
+  private static final String CASE_ALREADY_CANCELLED = "CASE_ALREADY_CANCELLED";
+
+  private static final String IGNORED_PAUSE_HH = "IGNORED_PAUSE_HH";
+
+  private static final ProcessorKey key = ProcessorKey.builder()
+      .actionInstruction(ActionInstructionType.PAUSE.toString())
+      .surveyName("CENSUS")
+      .addressType("HH")
+      .addressLevel("U")
+      .build();
+
+  @Autowired
+  private GatewayEventManager eventManager;
+
+  @Override
+  public ProcessorKey getKey() {
+    return key;
+  }
+
+  @Override
+  public boolean isValid(FwmtActionInstruction rmRequest, GatewayCache cache) {
+    try {
+      return rmRequest.getActionInstruction() == ActionInstructionType.PAUSE
+          && rmRequest.getSurveyName().equals("CENSUS")
+          && rmRequest.getAddressType().equals("HH")
+          && (cache != null && cache.existsInFwmt && cache.lastActionInstruction.equals("Cancel"));
```

Comment:
missing the address Level is U (I know its in the key fbut need to keep it consistenet)